### ISSUE TITLE
How to run Autotune?

### ISSUE THREAD

**Issue by Marco-rc** [br3ttb/Arduino-PID-AutoTune-Library#28]
Hello. For the autotune sketch, should I run it with the heater on or does it work even if I am not powering the heater? I am concerned the heater may get too hot while performing the autotune and something could melt above the desired temperatures. 
Here is the design I am using this with:
https://www.randseq.org/2020/02/3d-printer-filament-joiner.html?sc=1724725179594#c2170354465333568295

**Comment by UnexpectedMaker** [comment]
Yes the heater has to be on. How can it tune if it's not actually doing the thing you are wanting to tune for?

**Comment by Marco-rc** [comment]
Thank you.
By doing that I got the following parameters which seem to me out of range:
#define KP 0.62
#define KI 0.25
#define KD 0.39

As soon as I power the heater circuit on, the heater temperature starts
going up even if the "Heater off" is displayed in the monitor.
COuld it be that the MOSFET is connected the wrong way?
Marco Dominguez

**Comment by Marco-rc** [comment]
Thank you for your reply.

By having the heater on I got the following parameters which seem to me out of range:
#define KP 0.62
#define KI 0.25
#define KD 0.39

I noticed that ss soon as I power the heater circuit on, the heater temperature starts going up even if the "Heater off" is displayed in the monitor.
Could it be that the MOSFET is connected the wrong way?
